Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+@Library('shared-libraries') _
+pipeline{
+  agent {label 'javaClientLinuxPool'}
+  options {
+    checkoutToSubdirectory 'java-client-api'
+    buildDiscarder logRotator(artifactDaysToKeepStr: '7', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: '')
+  }
+  environment{
+    JAVA_HOME_DIR="/home/builder/java/openjdk-1.8.0-262"
+    GRADLE_DIR   =".gradle"
+    DMC_USER     = credentials('MLBUILD_USER')
+    DMC_PASSWORD = credentials('MLBUILD_PASSWORD')
+  }
+  stages{
+    stage('tests'){
+      steps{
+        copyRPM 'Release','10.0-9.4'
+        setUpML '$WORKSPACE/xdmp/src/Mark*.rpm'
+        sh label:'deploy project', script: '''#!/bin/bash
+          export JAVA_HOME=$JAVA_HOME_DIR
+          export GRADLE_USER_HOME=$WORKSPACE/$GRADLE_DIR
+          export PATH=$GRADLE_USER_HOME:$JAVA_HOME/bin:$PATH
+          cd java-client-api
+          ./gradlew mlDeploy
+        '''
+        sh label:'test', script: '''#!/bin/bash
+          export JAVA_HOME=$JAVA_HOME_DIR
+          export GRADLE_USER_HOME=$WORKSPACE/$GRADLE_DIR
+          export PATH=$GRADLE_USER_HOME:$JAVA_HOME/bin:$PATH
+          cd java-client-api
+          ./gradlew test  || true
```

Comment:
@SameeraPriyathamTadikonda I think this should just be marklogic-client-api:test - I don't think we need to run the tests for the other two projects yet, but maybe later.

Reply:
I changed to run functional tests to run after the PR merges.